docs/api/desktop-capturer.md

```diff
@@ -0,0 +1,70 @@
+# desktop-capturer
+
+The `desktop-capturer` is a renderer module used to capture the content of
+screen and individual app windows.
+
+```javascript
+// In the renderer process.
+var desktopCapturer = require('desktop-capturer');
+
+desktopCapturer.getSources({types: ['window', 'screen']}, function(sources) {
+  for (var i = 0; i < sources.length; ++i) {
+    if (sources[i].name == "Electron") {
+      navigator.webkitGetUserMedia({
+        audio: false,
+        video: {
+          mandatory: {
+            chromeMediaSource: 'desktop',
+            chromeMediaSourceId: sources[i].id,
+            minWidth: 1280,
+            maxWidth: 1280,
+            minHeight: 720,
+            maxHeight: 720
+          }
+        }
+      }, gotStream, getUserMediaError);
+      return;
+    }
+  }
+});
+
+function gotStream(stream) {
+  document.querySelector('video').src = URL.createObjectURL(stream);
+}
+
+function getUserMediaError(e) {
+  console.log('getUserMediaError');
+}
+```
+
+## Methods
+
+The `desktopCapturer` module has the following methods:
+
+### `desktopCapturer.getSources(options, callback)`
+
+* `options` Object, properties:
+  * `types` Array - An array of String that enums the types of desktop sources.
+    * `screen` String - Screen
```

Comment:
Our docs site doesn't support nesting list with more than 3 levels, can you put them into `types`'s description?